_data/checkParallelTool.yml

```diff
@@ -1,7 +1,8 @@
-description: Draws a line between on-curve points and BCPs and checks if they're parallel.
+description: Check if lines connecting BCPs and oncurves are parallel and edit BCPs by manipulating connection line.
 extensionName: Check Parallel Tool
 repository: https://github.com/jtanadi/CheckParallelTool
 extensionPath: CheckParallelTool.roboFontExt
 developer: Jesen Tanadi
 developerURL: http://jesentanadi.com
-tags: [contours, design helpers]
\ No newline at end of file
+icon: https://github.com/jtanadi/CheckParallelTool/blob/master/CheckParallelTool.roboFontExt/resources/checkParallelMechanicIcon.png
```

Comment:
could you change the path to the icon to the direct path?

https://raw.githubusercontent.com/jtanadi/CheckParallelTool/master/CheckParallelTool.roboFontExt/resources/checkParallelMechanicIcon.png

Reply:
All set. Thanks!